GeneratorAnalysis/Scripts/FlatTreeAnalyzer.cxx

```diff
@@ -357,6 +387,12 @@ void FlatTreeAnalyzer::Loop() {
             TrueCosOpeningAngleMuonTotalProtonPlot[genie_mode]->Fill(CosOpeningAngleMuonTotalProton,weight);
             TrueTransverseMomentumPlot[genie_mode]->Fill(TransverseMomentum,weight);
             TrueDeltaAlphaTPlot[genie_mode]->Fill(DeltaAlphaT,weight);
+	    TrueInvariantMassPlot[genie_mode]->Fill(InvariantMass, weight);
+
+            //Additional histo regardless of interaction mode
+            TrueInvariantMassPlot[genie_mode]->Fill(InvariantMass, weight);
```

Comment:
You are filling `TrueInvariantMassPlot` twice, which will lead to the output histograms to have 2x the events